bin/init.sh

```diff
@@ -225,9 +225,6 @@ fi
 
 # Donwload WebAssembly plugin files
 WASM_RELEASE_DIR=${ISTIO_ENVOY_LINUX_RELEASE_DIR}
-if [[ "$LOCAL_OS" == "Darwin" ]]; then
-  WASM_RELEASE_DIR=${ISTIO_ENVOY_MACOS_RELEASE_DIR}
-fi
```

Comment:
Why is Darwin being removed here?

Does https://github.com/istio/istio/pull/20372 address the same problem?

Reply:
These wasm files are linux only. So does the envoy binary built from proxy repo.

I initially added mac path to follow how envoy binary donwloading is done in this script.  But actually the macos path and the envoy binary downloaded to that path is not used at all. Its version stays at 1.0.2... It is always the envoy binary under linux path that is used in the final image built. So I see no reason to keep a macos path here for wasm. I am not sure but I don't think #20372 could solve the issue...

I think this script could use some clean up to just remove presence of macos from it since there is no plan and unlikely that we are going to ship an envoy binary built on macos.

Reply:
we use Mac for unit tests